classes/conta.ts

```diff
@@ -0,0 +1,38 @@
+import { Clientes } from "../interfaces/Cliente";
+
+export abstract class Conta {
+    protected saldo: number;
+
+    constructor(
+        public numeroConta: string,
+        public cliente: Clientes
+    ) {
+        this.saldo = 0;
+    }
+
+    public getSaldo(): number {
```

Comment:
Como saldo é um atributo da classe, acho que poderia usar um get mesmo. Ex.: get saldo(): number { return this.contaSaldo}
Assim o acesso da instância conta ao saldo poderia se dar como: conta.saldo

Reply:
Obrigada Ewelyn! Vou levar essas considerações na próxima entrega, obrigada <3